test/linter/test_psalm.vader

```diff
@@ -36,3 +36,8 @@ Execute(User provided options should be used):
   AssertLinter 'psalm',
   \ ale#Escape('psalm')
   \ . ' --language-server --my-user-provided-option my-value'
+
+Execute(The project path should be correct for composer.json file):
+  call ale#test#SetFilename('../test-files/php/with-composer/test.php')
+
+  AssertLSPProject ale#path#Simplify(g:dir . '/../test-files/php/with-composer')
```

Comment:
I'm using `AssertLSPProject` here but not sure if this is the best way to test this case.
I'll be more than happy to be corrected ;)